pymobiledevice3/services/dvt/instruments/process_control.py

```diff
@@ -40,6 +40,14 @@ def signal(self, pid: int, sig: int):
         self._channel.sendSignal_toPid_(MessageAux().append_obj(sig).append_obj(pid), expects_reply=True)
         return self._channel.receive_plist()
 
+    def memlimitoff(self, pid: int) -> list:
```

Comment:
please add a docstring for the returned list and supply the list with a specific type (e.g. `list[dict[str, str]]]`).
I know still most of current API don't use these typing, but I'm trying that at least all new code or modified one will have better typings